recipe/presentation/src/main/java/hu/tb/recipe/presentation/RecipeScreen.kt

```
@@ -0,0 +1,85 @@
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import hu.tb.presentation.theme.AppTheme
+
+@Composable
+fun RecipeScreen() {
+    
+}
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)